### PR TITLE
SDKS-2262 Provide suspend function for DeviceBindingCallback and DeviceSigningVerifierCallback

### PR DESCRIPTION
**Pull request by witrisna**
Provide suspend function for DeviceBindingCallback and DeviceSigningVerifierCallback

# JIRA Ticket

[Please link jira ticket here](https://bugster.forgerock.org/jira/browse/SDKS-2262)

# Description

Briefly describe the change and any information that would help speedup the review and testing.

# Definition of Done Checklist:

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Ensure backward compatibility.
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] Functional spec is written/updated.
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

### CODE REVIEW

File: forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/DeviceBindingCallbackFragment.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import org.forgerock.android.auth.callback.DeviceBindingCallback
+import org.forgerock.android.auth.ui.databinding.FragmentDeviceBindingCallbackBinding
+
+/**
+ * A simple [Fragment] subclass.
+ */
+class DeviceBindingCallbackFragment : CallbackFragment<DeviceBindingCallback?>() {
```
Comment (jeyanthanperiyasamy) [inline forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/DeviceBindingCallbackFragment.kt:23]: DeviceBindingCallback? - does this needs to be optional

File: forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/DeviceSigningVerifierCallbackFragment.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import org.forgerock.android.auth.callback.DeviceSigningVerifierCallback
+import org.forgerock.android.auth.ui.databinding.FragmentDeviceSigningVerifierCallbackBinding
+
+/**
+ * A simple [Fragment] subclass.
+ */
+class DeviceSigningVerifierCallbackFragment : CallbackFragment<DeviceSigningVerifierCallback?>() {
```
Comment (jeyanthanperiyasamy) [inline forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/DeviceSigningVerifierCallbackFragment.kt:23]: does this DeviceSigningVerifierCallback needs to optional ?

File: forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/DeviceBindingCallbackFragment.kt (hunk 1 above)
Comment (witrisna) [reply forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/DeviceBindingCallbackFragment.kt]: Remove optional

**Review by jeyanthanperiyasamy — CHANGES_REQUESTED** [review]
minor comments

**Review by jeyanthanperiyasamy — APPROVED** [review]

**Review by spetrov — APPROVED** [review]
👍🏻

**Pull request merged**